CONTRIBUTING.md

```diff
@@ -74,9 +74,11 @@ Run a single test
 `$ rake test TEST=path/to/test.rb TESTOPTS="--name=test_something"`
 
 Run tests against different Rails versions by setting the RAILS_VERSION variable
-and bundling gems.
+and bundling gems.  (save this script somewhere executable and run from top of AMS repository)
```

Comment:
nice. I just wrote this as something to paste into the console.  

I think a better solution to recommend something more robust would be to recommend using [wwtd](https://github.com/grosser/wwtd) and just use the current Ruby `wwtd --local`